Lib/idlelib/help.py

```diff
@@ -212,7 +212,9 @@ class HelpFrame(Frame):
     def __init__(self, parent, filename):
         Frame.__init__(self, parent)
         self.text = text = HelpText(self, filename)
-        self['background'] = text['background']
+        self.style = Style(parent)
+        self['style'] = 'helpframe.TFrame'
```

Comment:
Without this, the gray background for the menubutton extended down the entire length of the window.  I prefer the current box just around [TOC V].